### PR TITLE
Add Flask-Migrate support

### PR DESCRIPTION
**Pull request by armooo**
Add Flask-Migrate to manage database schema changes

Everything in migrations auto generated config from manage.py db init.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
+from flask.ext.migrate import Migrate, MigrateCommand
 from flask.ext.script import Manager
+
 from studygroup.application import create_app, db
 
+app = create_app()
 
-manager = Manager(create_app)
```
Comment (nedbat) [inline manage.py:5]: Is this right? We used to pass in the callable, but now pass in the result of calling it?

File: manage.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
+from flask.ext.migrate import Migrate, MigrateCommand
```
Comment (nedbat) [inline manage.py]: One of these days, someone should write a docstring for this file....

File: migrations/alembic.ini [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
```
Comment (nedbat) [inline migrations/alembic.ini:1]: Wow, I've never seen alembic before, they couldn't default a bunch of this stuff?

File: manage.py (hunk 1 above)
Comment (mmilkin) [inline manage.py:5]: it looks like flask_script takes either.

https://github.com/smurfix/flask-script/blob/v1.0/flask_script/__init__.py#L136-L139

**Comment by mmilkin** [comment]
Looks reasonable :+1:

**Pull request merged**